### PR TITLE
Fix light update failing.

### PR DESCRIPTION
**Pull request by vlebourl**
Light update fails if some features aren't available.

### CODE REVIEW

File: custom_components/tahoma/light.py [hunk 1]
```diff
@@ -144,10 +144,13 @@ def update(self):
...
         self._state = states.get(CORE_ON_OFF_STATE) == STATE_ON
-        self._brightness = states.get(CORE_LIGHT_INTENSITY_STATE)
 
-        self._hs_color = color_util.color_RGB_to_hs(
-            states.get(CORE_RED_COLOR_INTENSITY_STATE),
-            states.get(CORE_GREEN_COLOR_INTENSITY_STATE),
-            states.get(CORE_BLUE_COLOR_INTENSITY_STATE),
-        )
+        if CORE_LIGHT_INTENSITY_STATE in states:
+            self._brightness = states.get(CORE_LIGHT_INTENSITY_STATE)
```
Comment (iMicknl) [inline custom_components/tahoma/light.py:149]: What is the error you get? Because this should return None, which is the default value

cc @tetienne

Comment (vlebourl) [reply custom_components/tahoma/light.py:149]: yes but it goes through `color_util.color_RGB_to_hs` which doesn't accept NoneType.

Comment (iMicknl) [reply custom_components/tahoma/light.py:149]: Ok, makes sense, in that case you don't need this one for brightness, only for the hs_color.

File: custom_components/tahoma/light.py [hunk 2]
```diff
@@ -144,10 +144,13 @@ def update(self):
...
+
+        if CORE_RED_COLOR_INTENSITY_STATE in states:
```
Comment (iMicknl) [inline custom_components/tahoma/light.py:151]: Good spot, however I would check for all 3 values, since none of them can return None.

Comment (vlebourl) [reply custom_components/tahoma/light.py]:
```
[r, g, b] = [
            states.get(CORE_RED_COLOR_INTENSITY_STATE),
            states.get(CORE_GREEN_COLOR_INTENSITY_STATE),
            states.get(CORE_BLUE_COLOR_INTENSITY_STATE),
        ]
self._hs_color = (
            None if None in [r, g, b] else color_util.color_RGB_to_hs(r, g, b)
        )
```
this is better I think

Comment (vlebourl) [reply custom_components/tahoma/light.py]: I also removed the test from brightness.

**Pull request merged**